### DISCUSSION TITLE
Search Results - custom message in 'No results to display'

### DISCUSSION THREAD

**Discussion by Seykrits** [microsoft-search/pnp-modern-search#1384] [Ideas]
1.Add property pane setting to modify text for 'No results to display' is shown.
![image](https://user-images.githubusercontent.com/35454245/137050622-68fdee96-aef0-47f4-826c-2338824c243a.png)

2.At the moment 'No results to display' text always shows even before a search is performed.
Update: 
When 'Show blank if no result' toggle is off - 
The 'No results to display' text should only be displayed after a search is performed and no results returned, not on initial load where no default query is provided.
![image](https://user-images.githubusercontent.com/35454245/137051528-91904c23-b253-4251-aa68-7a0bfef68299.png)

**Comment by wobba** [comment]
As you can set this yourself in a custom template we don't want to overload with settings. Seems you are using v3 so change the code below to use your own string and not `@root.strings.NoResultMessage`. 

```
{{#unless showBlank}}
   <div class="template_noResults">{{@root.strings.NoResultMessage}}</div>
{{/unless }}
```
